charts/ks-devops/values.yaml

```diff
@@ -12,7 +12,7 @@ image:
   tools:
     repository: devops-tools
     tag: master
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Considering normal users, default pull policy should be `IfNotPresent`.

Reply:
Indeed, not good